source/grid/grid_tools.cc

```diff
@@ -2662,7 +2662,7 @@ namespace GridTools
     static inline std::bitset<3> lookup (const MATCH_T &)
     {
       // The 1D case is trivial
-      return 4; // [true ,false,false]
+      return 1; // [true ,false,false]
```

Comment:
I suppose this is an unrelated fix for something else. Do you happen to know how this worked before?